FogCreek/Glitch-Community
# Neighborly pull: fix resend invite on team pages

## Links
* https://neighborly-pull.glitch.me
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/118

## GIF/Screenshots:
![Screen Shot 2019-07-09 at 2 40 22 PM](https://user-images.githubusercontent.com/6620164/60914424-82b0f380-a257-11e9-9a2a-888fcf6f4cbf.png)

## Changes:
* fix bug where clicking `resend invite` on team page failed. 

## How To Test:
* create a team, invite a user, click on the user's face, click resend invite, invited user should have 2 invite emails in their inbox.
